packages/console/src/pages/SignInExperience/utilities.ts

```diff
@@ -81,11 +81,15 @@ export const compareSignInMethods = (
   before: SignInExperience,
   after: SignInExperience
 ): boolean => {
-  if (before.socialSignInConnectorIds.length !== after.socialSignInConnectorIds.length) {
+  if (before.socialSignInConnectorTargets.length !== after.socialSignInConnectorTargets.length) {
     return false;
   }
 
-  if (before.socialSignInConnectorIds.some((id) => !after.socialSignInConnectorIds.includes(id))) {
+  if (
+    before.socialSignInConnectorTargets.some(
+      (target) => !after.socialSignInConnectorTargets.includes(target)
+    )
+  ) {
```

Comment:
Want to make sure this method cares only about the existence of targets not the order of targets?

Reply:
YES